lib/resty/upstream/healthcheck.lua

```diff
@@ -213,65 +217,52 @@ local function check_peer(ctx, id, peer, is_backup)
         errlog("failed to create stream socket: ", err)
         return
     end
-
+    
```

Comment:
style: line trailing spaces? (BTW, such issues should be caught by tools like `lua-releng`).